contents/docs/integrate/client/js/index.mdx

```diff
@@ -305,6 +305,14 @@ posthog.reloadFeatureFlags()
 posthog.isFeatureEnabled('keyword', { send_event: false })
 ```
 
+#### Feature Flag Payloads
+Posthog v1.42.0 introduced payloads which allow you to retrieve a value that is associated with the matched flag. The value can be a
```

Comment:
let's mention the library version, instead of the posthog version. It's a lot less confusing I think 😅 (specially with the self-host deprioritisation!)

Reply:
oh, posthog-js is always latest, nevermind, only valid for node I guess